app/scripts/views/coppa/coppa-age-input.js

```diff
@@ -56,6 +56,22 @@ define(function (require, exports, module) {
       age.val(age.val().substr(0, 3));
     },
 
+    onKeyDown: function (event) {
+      // helper function to check for digit
+      function isKeyADigit (keyCode) {
+        return ((keyCode == 8) || // backspace
+        (keyCode >= 48 && keyCode <= 57) || // digit row
+        (keyCode >= 96 && keyCode <= 105)); // keypad
+      }
+
+      // submit on enter, force digit input
+      if (event.which === KeyCodes.ENTER) {
+        this.trigger('submit');
+      } else if (! isKeyADigitOrBackSpace(event.which)) {
```

Comment:
Your function above is `isKeyADigit`, here you call `isKeyADigitOrBackSpace`. Can you add each of the relevant keycodes to [key-codes.js](https://github.com/mozilla/fxa-content-server/blob/master/app/scripts/lib/key-codes.js#L9), for example I can imagine an entry `BACKSPACE: 8`. Instead of commenting `// backspace`, you can do `keyCode === KeyCodes.BACKSPACE` and the intent is clear. The digits would be useful over there too. I'm actually wondering if a helper method on KeyCodes, like `isDigit` would be useful, but that could be for a later day.